exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors=studentCount>mentorCount; // finish this statement
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let enoughSpaceInRoom=roomMaxCapacity===studentCount + mentorCount; // finish this statement
```

Comment:
```suggestion
let enoughSpaceInRoom = roomMaxCapacity === studentCount + mentorCount; // finish this statement
```

Reply:
Hi Luna, one suggestion I have is that you could make the code easier to read by adding spaces between operators and variables, etc. I have suggested an example change on line 13.